cli.js

```diff
@@ -14,7 +15,8 @@ const cli = meow(`
 	  211
 `);
 
-const input = cli.input[0];
+const input = cli.input[0] || cli.flags.pretty;
+const bytesFlag = cli.flags.pretty;
```

Comment:
It becomes necessary when you realize that you need a boolean flag for the ternary expression below AND that you need the `|| cli.flags.pretty` if the user inputs the file after the `--pretty` (`cli.input[0]` will not have any value in this case).